components/file-loader/awesome-file-loader.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+awesome.requireCSS(`${awesome.path}components/file-loader/awesome-file-loader.css`);
+
+(
+    function(){
+        const defaults={}
+
+        class Component extends HTMLElement{
+            createdCallback(){
+                awesome.mergeDataset(this,defaults);
+
+                this.innerHTML=`
+                    <input type='file' id='input-files' multiple>
+                    <p id='fileInfo'>
+
+                    </p>
+                    <p class='upload-text'>
+                        Select one or more files.
+                    </p>
+                `;
+            }
+
+            attachedCallback(){
+                this.addEventListener(
+                    'change',
+                    this.update
+                );
+            }
+
+            detachedCallback(){
+
+            }
+
+            attributeChangedCallback(key,oldValue,newValue){
+                this.createdCallback();
+            }
+
+            update(e){
+                let loadedFiles = document.getElementById('input-files').files;
+                document.querySelector('.upload-text').classList.add('hidden');
+                var txt='';
+
+                for(var i = 0; i < loadedFiles.length; i++){
+                    txt += "<br><strong>" + (i+1) + ". File</strong><br>";
+                    var file = loadedFiles[i];
+                    if ('name' in file) {
+                        txt += "name: " + file.name + "<br>";
+                    }
+                    if ('size' in file) {
+                        txt += "size: " + file.size + " bytes <br>";
+                    }
+                    if ('lastModifiedDate' in file) {
+                        txt += "lastModified: " + file.lastModifiedDate.toUTCString() + "<hr>";
+                    }
+                }
+                document.getElementById('fileInfo').innerHTML = txt;
```

Comment:
1. If this is called a file loader, you should be able to do something with the file other than display it. If this is meant to be a file info component, it would be best done as a file info component which only displays file content, not loads it. 
2. If this is infact a file loader, I think it would be good if this emitted some action event that could be used by actions later.
3. ***Isn't this kind of a duplicate of [awesome-file](https://github.com/RIAEvangelist/awesome-webcomponents/blob/master/components/file/awesome-file.js)***
4. use template strings, not string concats.